setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name = asdf_transform_schemas
+author = STScI
+author_email = help@stsci.edu
+license = BSD 3-Clause
+license_file = LICENSE
+description = ASDF schemas for transforms
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spacetelescope/asdf-transform-schemas
+
+[options]
+python_requires = >=3.6
+zip_safe = false
```

Comment:
Is there a reason the package is not zip-safe?

Reply:
I implemented the extension in "asdf classic" style, which means it returns a filesystem path to the location of the schema file in the installed package.  If the package were installed and loaded as a single zipped file, then the path wouldn't exist.